### DISCUSSION TITLE
Select default translation language

### DISCUSSION THREAD

**Discussion by dkophelyi** [directus/directus#8813] [Archive]
Is it possible to set the default translation language?
When I add multiple options, Directus shows the first entry based in alphabetical order. Can I change this behavior?
I would like to select a specific locale to be shown first when I add a new entry to a collection.

**Comment by johannesschobel** [comment]
it works with the "preferred language" for this specific user. However, this may be an issue, if you have users with various languages, i guess

**Comment by azrikahar** [comment]
#8203 should address your need here 👍

**Reply by superhp** [reply]
I see this #8203 is still open. Are there any plans to complete it? The possibility to set a default language for interface is really needed.

**Reply by azrikahar** [reply]
There's no plan for this specific PR yet per se, but reviewing/cleaning up PRs & releasing them when possible (tracked via [milestones](https://github.com/directus/directus/milestones)) is an ongoing process.
